src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,10 +198,7 @@ describe('ReactDOMComponent', function() {
 
       expect(console.error.argsForCall.length).toBe(1);
       expect(console.error.argsForCall[0][0]).toEqual(
-        'Warning: `span` was passed a style object that has previously been ' +
-        'mutated. Mutating `style` is deprecated. Consider cloning it ' +
-        'beforehand. Check the `render` using <span>. Previous style: ' +
-        '{fontSize: NaN}. Mutated style: {fontSize: NaN}.'
+        'Warning: `NaN` is an invalid value for the `fontSize` css style property'
```

Comment:
Let's do this (mostly what I said but updated for correctness):
- Leave this test but we need to update the `1` on line 199 to a `2` (to account for the warning you're adding), and then update the `argsForCall[0][0]` to `argsForCall[1][0]`
- Add a new test explicitly for the new warning in CSSPropertyOperations-test
- Ignore what @jimfb said (it's relevant but not to you right now)